mlflow_databricks_artifacts/protos/scalapb/scalapb.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto2";
+
```

Comment:
Because the proto definitions for ACLed artifacts APIs are not available in older versions of MLflow, we include `databricks_artifacts.proto` and its dependencies (`databricks.proto` / `scalapb.proto`)